src/pages/organisationUnits/New.tsx

```diff
@@ -3,41 +3,22 @@ import { FormBase } from '../../components'
 import { DefaultNewFormContents } from '../../components/form/DefaultFormContents'
 import { SECTIONS_MAP, useOnSubmitNew, validate } from '../../lib'
 import {
-    FormValues,
     initialValues,
     OrganisationUnitFormField,
     organisationUnitSchema,
 } from './form'
+import { OrgUnitFormValues } from './Edit'
 
-const formatFormValues: (values: FormValues) => Record<string, unknown> = (
-    values
-) => {
-    return {
-        ...values,
-        geometry:
-            values.geometry?.longitude && values.geometry?.latitude
-                ? {
-                      type: 'Point',
-                      coordinates: [
-                          values.geometry?.longitude,
-                          values.geometry?.latitude,
-                      ],
-                  }
-                : undefined,
-        attributeValues: values.attributeValues.filter(({ value }) => !!value),
-    }
-}
 const section = SECTIONS_MAP.organisationUnit
 
 export const Component = () => {
     return (
         <FormBase
             onSubmit={useOnSubmitNew({
                 section,
-                valueFormatter: formatFormValues,
             })}
-            initialValues={initialValues as FormValues}
-            validate={(values: FormValues) => {
+            initialValues={initialValues as OrgUnitFormValues}
+            validate={(values: OrgUnitFormValues) => {
```

Comment:
type should be inferred from `initialValues` above. No need to type this (and it's not actually `OrgUnitFormValues` here - it should be `typeof initialValues`.